### DISCUSSION TITLE
Dockge - Volumes not Found?

### DISCUSSION THREAD

**Discussion by gschmucker95ski** [louislam/dockge#34] [Ask for Help]
Hello,

I have the problem that my Gitlab Instance won't migrate to dockge. 
I use Dockge via docker-compose....

Here is my docker-compose.yml for dockge:
![image](https://github.com/louislam/dockge/assets/150692889/4f66680b-eec6-4baa-8fb6-1657b5df3033)

Maybe I'm doin something wrong in the volumes configuration?
### Gitlab Compose in Dockge

![image](https://github.com/louislam/dockge/assets/150692889/c465d6cb-2c02-4596-9794-53f02bbad0bc)

![image](https://github.com/louislam/dockge/assets/150692889/8e7aeebb-bbee-49bb-9b6b-7c41fdf9b788)

**Comment by louislam** [comment]
Can you spin up your gitlab using docker-compose?

```bash
cd /my-stacks/gitlab
docker compose up
```

**Comment by gschmucker95ski** [comment]
That works fine, but creating, editing or start/stop/restart over the Frontend it brokes :/

**Comment by gschmucker95ski** [comment]
The issue was oddly resolved after renaming the "my-stacks" folder to "mystacks." Renaming-Changes were also made in the docker-compose.yaml file.


The case can now be closed.
